yoti-sdk-api/src/main/java/com/yoti/api/client/identity/constraint/PreferredSources.java

```diff
@@ -0,0 +1,39 @@
+package com.yoti.api.client.identity.constraint;
+
+import java.util.List;
+
+import com.yoti.api.client.identity.policy.WantedAnchor;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class PreferredSources {
```

Comment:
I assume this will never be used for de-serialization? Cause as it stands, they don't support it. They only are good for serialisation.